Prioritize/Java/Multi-Module/Maven/github-action.yml

```diff
@@ -13,18 +13,22 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up JDK 1.11
-      uses: actions/setup-java@v1
+    - name: Set up JDK
+      uses: actions/setup-java@v2
       with:
-        java-version: 1.11
+        java-version: '8'
+        distribution: 'adopt'
     - name: Build with Maven
       run: mvn clean install -DskipTests=true
     - name: WhiteSource Unified Agent Scan
       env:
-        APIKEY: ${{secrets.APIKEY}}
+        GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+        WS_APIKEY: ${{secrets.APIKEY}}
+        WS_USERKEY: ${{secrets.USERKEY}}
       run: |
         cat <<EOF > eua.config
-        apiKey=$APIKEY
+        apiKey=$WS_APIKEY
+        userKey=$WS_USERKEY
```

Comment:
Remind me please, why are we using both env vars *and* a config file in the same scan?
(namely, the apiKey and userKey are completely irrelevant if you already declare the env variables)

Reply:
because the multimodule analyzer does not recognize env variables.  I just tested it again using only env variables and it throws the following error - Error:  01:46:30 - Effective usage analysis could not run, the file DEFAULT does not exist.
See[this example]( https://github.com/samq-ghdemo/Gradle-multimodule/runs/2706924863?check_suite_focus=true)